scripts/build/concatenate-javascript.sh

```diff
@@ -48,3 +52,8 @@ if [ -z "$ARGV_ENTRY_POINT" ] || [ -z "$ARGV_OUTPUT" ]; then
 fi
 
 "$BROWSERIFY" "$ARGV_ENTRY_POINT" --node --outfile "$ARGV_OUTPUT"
+
+if [ "$ARGV_MINIFY" == "true" ]; then
+  uglifyjs --compress --output "$ARGV_OUTPUT.MIN" -- "$ARGV_OUTPUT"
+  mv "$ARGV_OUTPUT.MIN" "$ARGV_OUTPUT"
```

Comment:
Sorry, I guess that was a bit of a confusing comment.
I've just checked, and I now see that `uglifyjs` actually outputs a file - given that it takes a directory as its input-argument, I'd mistakenly assumed that it also outputted a directory, and that the `$ARGV_OUTPUT.MIN` output-directory was then being renamed over the `$ARGV_OUTPUT` input directory ;-)
Does `uglifyjs --compress --output "$ARGV_OUTPUT/$ARGV_OUTPUT.MIN" -- "$ARGV_OUTPUT"` not work correctly then?

Also, what is it that actually makes use of the `"$ARGV_OUTPUT/$ARGV_OUTPUT.MIN"` file?